tests-scan

```diff
@@ -102,18 +102,19 @@ def main():
 
 
 # Prepare a human readable output
-def tests_human(priority, name, number, revision, ref, context, base, original_base, repo, bots_ref):
+def tests_human(priority, name, number, revision, ref, context, base, original_base, repo, github_context, bots_ref):
     if not priority:
         return
     try:
         priority = int(priority)
     except (ValueError, TypeError):
         pass
-    return "{name:11} {context:25} {revision:10} {priority:2}{repo}{bots_ref}{branches}".format(
+    return "{name:11} {context:25}{github_context} {revision:10} {priority:2}{repo}{bots_ref}{branches}".format(
         priority=priority,
         revision=revision[0:7],
         context=context,
         name=name,
+        github_context=github_context == "master" and "" or (" " + github_context),
```

Comment:
github_context will never be "master". It is literally the name of the test in github with all parts.

Reply:
TBH I didn't look too much into what it means, but I started with just printing the value, and `bots/tests-scan -vd` just showed "master" everywhere. I just sent PR #13561 which has a bots PR test, but it doesn't show that.

So I adjusted this PR to simply accept and ignore the argument for now.

Reply:
Argh, I had the argument in the wrong place, I really saw bots_ref, sorry.